runtime/compiler/optimizer/TreeLowering.cpp

```diff
@@ -80,12 +78,22 @@ TR::TreeLowering::moveNodeToEndOfBlock(TR::Block* const block, TR::TreeTop* cons
          {
          TR::TreeTop* nextTT = iterTT->getNextTreeTop();
          TR::ILOpCodes op = iterTT->getNode()->getOpCodeValue();
+         bool opCodeCondition = false;
```

Comment:
A more descriptive name than `opCodeCondition` might be appropriate - `isStoreOp` or `isStoreOpOfInterest`, perhaps?